5_6-Jan/alternatingChar.js

```diff
@@ -0,0 +1,11 @@
+let alternatingCharacters = (arrOfInputStrings = []) => {
+  return arrOfInputStrings.map(input => {
+    let minimumDeletionCount = 0;
+    for (const [index, elem] of input.split('').entries()) {
```

Comment:
A lot of things happening on a single line, use a variable to store data instead